Update R&D organization and AORs (#1178)
mattermost/mattermost-handbook: operations/research-and-development/organization/README.md

* **[Server](/operations/research-and-development/organization/server.md)**
* **[QA](/operations/research-and-development/organization/qa.md)**
* **[Tech Writing](/operations/research-and-development/organization/tech_writing.md)**

## Growth

The Growth division develops solutions and features across the spectrum that allow us to reach more users and customers.

* **[Growth](/operations/research-and-development/organization/growth.md)**
* **[Self-Serve](/operations/research-and-development/organization/self-serve_subscriptions.md)**

## Data Engineering

The Data Engineering division builds and supports the data infrastructure and analysis tools.

* **[Data Engineering](/operations/research-and-development/organization/data_engineering.md)**

## Cloud

The Cloud division is responsible for the delivery and operations of Mattermost products across SaaS and self-hosted deployment options.

* **[Release DevOps](/operations/research-and-development/organization/release_devops.md)**
* **[Cloud Platform](/operations/research-and-development/organization/cloud_platform.md)**
* **[Site Reliabilty Engineering](/operations/research-and-development/organization/sre.md)**

## Security

> The Security division is responsible for both product and security operations, in addition to security certifications.

Reviewer: ```suggestion
The Security division is responsible for the implementation and monitoring of the company's security program.
```

> * **[Product Security](/operations/research-and-development/organization/product_security.md)**

Reviewer: ```suggestion
* **[Governance, Risk, and Compliance (GRC)](/operations/research-and-development/organization/grc.md)**
* **[Product Security](/operations/research-and-development/organization/product_security.md)**
```

# R&D Organization

The source of truth for the R&D org structure and team membership is [this spreadsheet](https://docs.google.com/spreadsheets/d/1lH8QIjQGEoGospDUdVs_LQ_i2b82I1ce6W7z18vhPTQ/edit#gid=1820415931).

R&D is split into multiple divisions, detailed below. Areas of responsibility (AORs) for each team are linked.

## Product

The Product division builds and ships the majority of anything product related.

### Core Product

> The Core Product sub-division is responsible for building the features and systems of the core product.

Reviewer 2: Maybe this sounds better once the teams are no longer named Core, but right now it feels like trying to define  a word with itself.
But I don't have a better suggestion for it. :/

Author: Yeah, I struggled with that a bit. "The core team builds the core" is a little unnecessary but it also felt weird not providing any description.
